### PR TITLE
OpenVINO Add precision type to inputs and int4 support

### PR DESCRIPTION
**Pull request by Vladimir9812**
Hi, @lutzroeder, in this PR has been added support for int4 and uint4 layer precisions and have been enabled precisions for layer inputs

### CODE REVIEW

File: source/openvino.js
Comment (lutzroeder) [inline source/openvino.js:132]: Lookup map and `find` can be inefficient.

File: source/openvino.js
Comment (lutzroeder) [inline source/openvino.js:135]: * What exactly is this code doing?
* Can the `precision` of the output always assumed to be the same as the input.
* `input` contains source data that should be copied instead of modified.

**Review by lutzroeder — CHANGES_REQUESTED** [review]

File: source/openvino.js
Comment (Vladimir9812) [reply source/openvino.js:132]: What lookup map can you suggest?

File: source/openvino.js
Comment (Vladimir9812) [reply source/openvino.js:135]: Initially in XML file inputs don't have precision property, unlike outputs. So, I try to match inputs and outputs according to the edges map and assign precisions to inputs.
Yes, precision to the output always assumed to be the same as the input.

File: source/openvino.js
Comment (lutzroeder) [reply source/openvino.js:132]: `const layers = new Map(net.layers.map((entry) => [ entry.id, entry ]))` and `layers.get(outputLayerId))`.

File: source/openvino.js
Comment (Vladimir9812) [reply source/openvino.js:130]: Thank you, @lutzroeder, I will fix all your comments tomorrow during the day

File: source/openvino.js
Comment (Vladimir9812) [reply source/openvino.js:948]: This is a model with int4 for test
[int4-model.zip](https://github.com/lutzroeder/netron/files/6259225/int4-model.zip)

**Comment by Vladimir9812** [comment]
@lutzroeder, I don't have any information about correctly decoding int4 dataType. So I've added a stub so far, but it is a temporary decision. I corrected all the other comments and PR is waiting for a second review round

File: source/openvino.js
Comment (lutzroeder) [reply source/openvino.js:137]: @Vladimir9812 "assumed" to be the same as in this code makes this assumption but it could be wrong or is this always the case for all OpenVINO models?

File: source/openvino.js
Comment (lutzroeder) [reply source/openvino.js:967]: Can you provide a smaller model? 10 MB is a lot of a test file to test a data type.

File: source/openvino.js
Comment (Vladimir9812) [reply source/openvino.js:137]: @lutzroeder, sorry for my long answers...Yes, it is true for all OpenVINO models.
Thank you for merge. Can I find a smaller int4 model and attach it to this PR?

Comment (lutzroeder) [reply source/openvino.js:137]: Yes, that would be great.

Comment (Vladimir9812) [reply source/openvino.js:137]: [simple_model_int4.zip](https://github.com/lutzroeder/netron/files/6278507/simple_model_int4.zip)

**Pull request closed without merging**
